lib/wellies_web/parser/open_weather.ex

```diff
@@ -0,0 +1,47 @@
+defmodule WelliesWeb.OpenWeatherParser do
+
+  def get_current_forecast(body) do
+     body
+     |> get_weather_list()
+     |> List.first()
+  end
+
+  def temperatures_grouped_by_date(body) do
+    body
+    |> get_weather_list()
+    |> clean_date_field()
+    |> group_by_date()
+  end
+
+  defp get_weather_list(body), do: Map.get(body, "list")
+
+  defp clean_date_field(element), do: Enum.map(element, fn x -> update_date(x) end)
```

Comment:
In these kind of cases, you can refactor this to simply reference the used function, in the following way:

```
defp clean_date_field(element), do: Enum.map(element, &update_date/1)
```

Check this resource: https://culttt.com/2016/05/09/functions-first-class-citizens-elixir/

Reply:
The same applies to all the below :)